### PR TITLE
[ja] Localize Chaos Engineering into Japanese

### LINKED ISSUE #2819: [ja] Localize Chaos Engineering into Japanese
Localize https://glossary.cncf.io/chaos-engineering/ into Japanese

### PR DESCRIPTION
**Pull request by Okabe-Junya**
### Describe your changes

Localize Chaos Engineering into Japanese

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2819 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
The target page on the deploy preview:

- https://deploy-preview-2986--cncfglossary.netlify.app/ja/chaos-engineering/

File: content/ja/chaos-engineering.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+インフラストラクチャー、プラットフォーム、あるいは([マイクロサービス](/ja/microservices-architecture)ベースの)アプリケーションの他の動作部分のように、アプリケーションの停止につながる可能性のある複数の側面が関与しています。
+本番環境に新機能を高頻度でデプロイすることは、高い確率でダウンタイムと重大なインシデントをもたらし、ビジネスに重大な影響を及ぼす可能性があります。
+
+## どのように役に立つのでしょうか
+
+カオスエンジニアリングは、回復力の要件を満たすための技術です。
+インフラストラクチャー、プラットフォーム、そしてアプリケーションの障害に対する回復力を達成するために使用されます。
+カオスエンジニアは、アプリケーション、インフラストラクチャー、あるいはプラットフォームが自己修復でき障害が顧客に目立った影響を与えないことを検証するために、プロアクティブにランダムな障害を注入するカオス実験を行います。
+カオス実験の目的は、盲点(例えば、モニタリングや自動スケーリング技術)を発見し、重大なインシデントの最中にチーム間のコミュニケーションを改善することです。
+このアプローチは、特に本番環境において、複雑なシステムにおける回復力とチームの自信を高めるのに役立ちます。
```
Comment (naonishijima) [inline content/ja/chaos-engineering.md:24]: こちらが人が対象なので自信で良いと思います。

File: content/ja/chaos-engineering.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: カオスエンジニアリング
+status: Completed
+category: コンセプト
+tags: ["方法論", "", ""]
+---
+
+カオスエンジニアリング(CE)とは、本番環境の[分散システム](/ja/distributed-systems/)に実験する学問分野です。
+分散システムが乱雑で予期せぬ状況に耐えるシステムの能力に自信を持つことを目指します。
```
[the remark this answers is not in the document]
Comment (Okabe-Junya) [reply content/ja/chaos-engineering.md]: ありがとうございます。確かにご指摘の通り、「信頼」の方が適切に感じました

https://github.com/cncf/glossary/pull/2986/commits/8939117499924137f993d9ad81745337041de141 で修正しました

**Review by naonishijima — APPROVED** [review]
LGTM

**Comment by Okabe-Junya** [comment]
ありがとうございます〜！

File: content/ja/chaos-engineering.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+category: コンセプト
+tags: ["方法論", "", ""]
+---
+
+カオスエンジニアリング(CE)とは、本番環境の[分散システム](/ja/distributed-systems/)に実験する学問分野です。
+分散システムが乱雑で予期せぬ状況に耐えるシステムの能力に対する信頼を築くことを目指します。
+
+## 解決すべき問題はなんですか
+
+[SRE](/ja/site-reliability-engineering/)や[DevOps](/ja/devops/)の実践は、プロダクトの回復力と信頼性を高める技術に焦点を当てています。
```
Comment (kaitoii11) [inline content/ja/chaos-engineering.md:13]:
```suggestion
[SRE](/ja/site-reliability-engineering/)や[DevOps](/ja/devops/)の実践は、プロダクトの回復力と[信頼性](/ja/reliability/)を高める技術に焦点を当てています。
```

**Review by kaitoii11 — CHANGES_REQUESTED** [review]

**Comment by Okabe-Junya** [comment]
すいません、完全に抜けていました。。

**Pull request merged**